netlify.toml

```diff
@@ -62,6 +66,45 @@ HUGO_ENABLEGITINFO = "true"
   to = "https://raw.githubusercontent.com/projectcontour/contour/:splat/examples/render/contour.yaml"
   status = 302
 
+# Redirect /quickstart/contour-gateway.yaml to the example Contour-Gateway manifest that matches :latest.
+#
+# kubectl apply https://projectcontour.io/quickstart/contour-gateway.yaml
+[[redirects]]
+from = "/quickstart/contour-gateway.yaml"
+to = "https://raw.githubusercontent.com/projectcontour/contour/release-1.24/examples/render/contour-gateway.yaml"
+status = 302
+
+# Redirect versioned quickstarts so that they can easily be referenced by
+# users or for upgrade testing.
+#
+# kubectl apply https://projectcontour.io/quickstart/v1.4.0/contour-gateway.yaml
+[[redirects]]
+  from = "/quickstart/*/contour-gateway.yaml"
+  to = "https://raw.githubusercontent.com/projectcontour/contour/:splat/examples/render/contour-gateway.yaml"
+  status = 302
```

Comment:
I added this one for consistency since it was missing, otherwise this should just be moving things around to make easier to maintain going fwd